pkg/plugins/golang/v4/scaffolds/internal/templates/hack/boilerplate.go

```diff
@@ -106,8 +106,8 @@ Copyright {{ .Year }}.
 {{ index .Licenses .License }}*/`
 
 var knownLicenses = map[string]string{
-	"apache2": apache2,
-	"none":    "",
+	"apache2":   apache2,
+	"copyright": "",
```

Comment:
We cannot have any copyright withing the value of none.
I think we might need to check if is possible to pass here: boilerplateTemplate

Reply:
Actually I think the 'const copyright' == “”（an empty string) is enough. the ` {{ index .Licenses .License }}*/` ` will just be an empty string.